server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,23 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class ExamEntity {
+    @Id
+    private Long Id;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    @Column(unique = true)
+    private String name;
+    private int weight;
+
+    @OneToMany(mappedBy = "exam")
+    private Set<CandidateEvaluationGridEntity> candidateEvaluationGrids;
+    @ManyToMany(mappedBy = "exams")
+    private Set<SkillEntity> skills;
```

Comment:
c'est unidirectionnelle la
```suggestion
    @ManyToMany
    private Set<SkillEntity> skills;
```